### PR TITLE
net-misc/netsed: revbump, EAPI 8, fix test

### PR DESCRIPTION
**Pull request by scardracs**
Closes: https://bugs.gentoo.org/740806

Signed-off-by: Marco Scardovi<mscardovi@icloud.com>

### CODE REVIEW

File: net-misc/netsed/netsed-1.3-r1.ebuild [hunk 1]
```diff
@@ -10,6 +10,10 @@ SRC_URI="http://silicone.homelinux.org/release/netsed/${P}.tar.gz"
 LICENSE="LGPL-2"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+DEPEND="test? ( dev-lang/ruby )"
```
Comment (thesamesam) [inline net-misc/netsed/netsed-1.3-r1.ebuild:16]: BDEPEND, as we're running Ruby

File: net-misc/netsed/netsed-1.3-r1.ebuild [hunk 2]
```diff
@@ -1,7 +1,7 @@
-# Copyright 1999-2020 Gentoo Authors
+# Copyright 2022 Gentoo Authors
```
Comment (juippis) [inline net-misc/netsed/netsed-1.3-r1.ebuild:1]: Why change this copyright header? Everything looks good otherwise.

**Review by juippis — APPROVED** [review]

File: net-misc/netsed/netsed-1.3-r1.ebuild (hunk 2 above)
Comment (scardracs) [reply net-misc/netsed/netsed-1.3-r1.ebuild:1]: I did not, pkgdev changed it aoutmatically 😕

Comment (scardracs) [reply net-misc/netsed/netsed-1.3-r1.ebuild:1]: I did not 😕 I find sometimes pkgdev did it during pkgdev commit

**Pull request closed without merging**